backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1758,3 +1758,61 @@ databaseChangeLog:
             columns:
               - column:
                   name: test_result_delivery
+  - changeSet:
+      id: add-no-phi-views
+      author: josh@skylight.digital
+      comment: Add views for some tables that omit columns containing PHI.
+      changes:
+        - createView:
+            viewName: api_user_no_phi_view
+            remarks: A subset of the api_user table with columns containing PHI removed.
+            replaceIfExists: true
+            fullDefinition: false
+            selectQuery: SELECT internal_id, created_at, updated_at, last_seen, is_deleted FROM ${database.defaultSchemaName}.api_user
+        - createView:
+            viewName: provider_no_phi_view
+            remarks: A subset of the provider table with columns containing PHI removed.
+            replaceIfExists: true
+            fullDefinition: false
+            selectQuery: SELECT internal_id, created_at, created_by, updated_at, updated_by, is_deleted, provider_id FROM ${database.defaultSchemaName}.provider
```

Comment:
Yeah I'll fix the widths, I meant to go back and clean it up and forgot. As far as the requirements, this is another place I'd like @TimBest's input. It does seem like we shouldn't be letting people look up PII either.

Reply:
I think it's an open question whether user PII is something we should have access to through metabase, since it's a support tool and there's no other good way to link up a real-world user of the application with records in the DB. Not the most pressing part of this ticket, though, for sure.